### PR TITLE
chore: update GAE samples to Go 1.15

### PR DESCRIPTION
**Pull request by engelke**

### CODE REVIEW

File: appengine/go11x/cloudsql/go.mod [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module cloudsql
 
-go 1.11
+go 1.15
```
Comment (tbpg) [inline appengine/go11x/cloudsql/go.mod:3]: Please keep the `go.mod` versions the same. They specify the minimum version of Go for the module.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#go-version-in-gomod-files (new from a couple weeks ago).

**Review by tbpg — CHANGES_REQUESTED** [review]
Thanks for doing this!

File: appengine/go11x/cloudsql/go.mod (hunk 1 above)
Comment (engelke) [reply appengine/go11x/cloudsql/go.mod]: Sorry, I remember you told me this last time, but I forgot. This is now done.

**Review by tbpg — APPROVED** [review]
Thanks!

**Pull request merged**